aioaws/s3.py

```diff
@@ -181,6 +194,7 @@ def signed_upload_url(
         size: int,
         content_disp: bool = True,
         expires: Optional[datetime] = None,
+        acl: ACLType = "public-read",
```

Comment:
Best if we can keep the default unchanged.

Reply:
Hey, thanks for the quick response.

I took the example from: [Example: Browser-Based Upload using HTTP POST](https://docs.aws.amazon.com/AmazonS3/latest/API/sigv4-post-example.html) and [POST Policy](https://docs.aws.amazon.com/AmazonS3/latest/API/sigv4-HTTPPOSTConstructPolicy.html)

Will add some thanks for sure, do you mean set the `acl` to `None` and only use it in case if set? Okay, no problem

Reply:
And the (ACL Definition)[https://docs.aws.amazon.com/AmazonS3/latest/userguide/acl-overview.html]